tasks/main.yml

```diff
@@ -1,20 +1,63 @@
 ---
+- name: Update repositories cache and install java
+  apt:
+    name: default-jdk
+    update_cache: yes
+  sudo: yes
+
+- name: Create destination folder
+  file:
+    path: "{{ dest }}"
+    state: directory
+    owner: "{{ server_user }}"
+    group: "{{ server_group }}"
+    mode: 0755
+  sudo: yes
+  sudo_user: "{{ server_user }}"
+
 - name: Download spark-hadoop
   get_url:
     url: https://d3kbcqa49mib13.cloudfront.net/spark-2.1.1-bin-hadoop2.7.tgz
```

Comment:
please turn this into a variable inside `spark/vars/main.yml`. In case of a new version release for spark or a different tgz spark file to download.